backend/ibutsu_server/tasks/importers.py

```diff
@@ -124,6 +140,10 @@ def run_junit_import(import_):
                 "params": {},
                 "source": ts.get("name"),
             }
+
+            # Extend the result metadata with import metadata
+            if metadata:
+                result_dict["metadata"].update(metadata)
```

Comment:
We should also set `component`/`env` from the metadata for the junit import (both on the result and the run)

Reply:
Good call on these changes, they're added now.

Reply:
These and the ones you mentioned above.